0. Introduction/0.2 Principles.md

```diff
@@ -39,54 +39,54 @@ Entanglements (vertical or horizontal) also prevent engineers in any system (esp
+The Standard enforces the idea of airplane mode. Where engineers can stand up their entire infrastructure on their local machine without needing or having any network connection, this principle goes heavily against principles such as Cloud-Native applications, where a given system cannot stand and run without cloud infrastructure.
 
-The Standard also encourages it's adapters to develop the proper tooling to bridge the gap between cloud infrastructural components and local components such as queues, event hubs and any other tools to make it easily testable and modifiable.
+The Standard also encourages its adopters to develop the proper tooling to bridge the gap between cloud infrastructural components and local components such as queues, event hubs, and other tools to make it easily testable and modifiable.
 
 ## 0.2.6 No Toasters
-The Standard shall be taught man to man not machine to man. There should be no stylecops or analyzers implemented to force people into following The Standard. It should be driven by passion in the heart and conviction in the mind. The Standard should be taught by people to people. It's meant to foster an engineering culture of open discussions, conviction and understanding.
+The Standard shall be taught man to man, not machine to man. No stylecops or analyzers should be implemented to force people into following The Standard. It should be driven by passion in the heart and conviction in the mind. The Standard should be taught by people to people. It's meant to foster an engineering culture of open discussions, conviction and understanding.
 
 ## 0.2.7 Pass Forward
-The Standard shall be taught at no cost. As it arrived to you at no cost. It should be passed forward to the next engineer also at no cost. Regardless of their financial, social or educational status. The Standard is pure knowledge given by the selfless to the selfless. There shall be no profiteering off of it, neither shall it be a reason for someone to belittle others or make them feel less. Teaching The Standard for profit violates The Standard, and denies it's beneficiary any further guidance from The Author.
+The Standard shall be taught at no cost as it arrived to you at no charge. It should be passed forward to the next engineer also at no cost, regardless of their financial, social, or educational status. The Standard is pure knowledge given by the selfless to the selfless. There shall be no profiteering off of it; neither shall it be a reason for someone to belittle others or make them feel less. Teaching The Standard for profit violates it and denies the beneficiary (the violator) any further guidance from The Author.
```

Comment:
@dlandi keep this as `no cost` instead of `no charge` it rhymes better.

Reply:
done